doc/04-Getting-started.md

```diff
@@ -28,6 +28,67 @@ this is an advanced task you should not tackle at the early beginning.
 
 Take some time to really understand how to work with Icinga Director first.
 
+First steps on the Frontend
+====================================
+Create a host
+------------------
+
```

Comment:
The description of the "Name" field explicitly suggest to use meaningful names for templates and argues strictly against `generic-host` and similar. We should not continue to suggest bad practices in our documentation.